src/cascade/input_data/db/csmr.py

```diff
@@ -83,7 +83,7 @@ def get_csmr_data(execution_context):
 
     csmr = csmr[csmr["val"].notnull()]
 
-    csmr.rename(columns={"val": "mean"})
+    csmr = csmr.rename(columns={"val": "mean"})
```

Comment:
How did this get by us? That was bad. And does fixing this mean we expect "val" later? What's downstream?

Reply:
Yeah, that's pretty scary. It was getting implicitly renamed int the freeze-unfreeze operation because :bomb: